addons/website/__init__.py

```diff
@@ -37,8 +37,3 @@ def rem_website_id_null(dbname):
             ]).unlink()
 
     cr.postcommit.add(partial(rem_website_id_null, cr.dbname))
```

Comment:
Oh, indeed ! The following happens: the theme images go from this in 14.0
```
        'static/description/beauty_description.png',
        'static/description/beauty_screenshot.jpg',
```
to this in 15.0:
```
        'static/description/beauty_description.jpg',
        'static/description/beauty_screenshot.jpg',
```
Therefore the last created image becomes `beauty_description.jpg` instead of the screenshot.
I will do the following: if any image needs to be create => create all images. This way they will remain in the correct order.